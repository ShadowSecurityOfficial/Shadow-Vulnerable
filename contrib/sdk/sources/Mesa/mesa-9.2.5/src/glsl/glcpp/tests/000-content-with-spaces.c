version https://git-lfs.github.com/spec/v1
oid sha256:8c8a51a58d33c7754dca148cbcd336a2bbedec87b6ef4c1c6cd3e195d10f642e
size 22
