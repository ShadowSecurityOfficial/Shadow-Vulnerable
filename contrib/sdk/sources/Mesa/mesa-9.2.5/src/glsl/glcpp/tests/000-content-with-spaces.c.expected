version https://git-lfs.github.com/spec/v1
oid sha256:2cae39d16f1f374380b3672d26962085fd28d128423c183507e895c021fa3504
size 21
